### ISSUE TITLE
[Bug Report] InvalidProtocolBufferException due to version of security-crypto

### ISSUE THREAD

**Issue by efguydan** [naver/naveridlogin-sdk-android#87]
Due to the version of security crypto being used, this crash occurs on some devices

### Stack trace
```
Caused by com.google.crypto.tink.shaded.protobuf.InvalidProtocolBufferException
Protocol message contained an invalid tag (zero).
com.google.crypto.tink.shaded.protobuf.GeneratedMessageLite.parsePartialFrom (GeneratedMessageLite.java:1566)

androidx.security.crypto.EncryptedSharedPreferences.create (EncryptedSharedPreferences.java:120)
com.navercorp.nid.oauth.EncryptedPreferences.migration (EncryptedPreferences.kt:120)
com.navercorp.nid.oauth.EncryptedPreferences.setContext (EncryptedPreferences.kt:44)
com.navercorp.nid.NaverIdLoginSDK.initialize (NaverIdLoginSDK.java:56)
```

A workaround is to downgrade the version to alpha01 or switch away from this dependency.
https://issuetracker.google.com/issues/210008257

**Comment by orafaaraujo** [comment]
Please, any updates on this? 

Is it possible to replace this dependency or not use an alpha/beta version? They have the version [1.0.0 release.](https://mvnrepository.com/artifact/androidx.security/security-crypto?repo=google) 

We have been forcing the downgrade since mid-2022 and would like to remove this hack from our code. 

Thank you

**Comment by DayeonLee-0** [comment]
Hello, @efguydan 

I'm an Android Developer for Naver Id Login SDK.

This issue has been patched in SDK v5.8.0.

Please refer to the development.

Thank you.
